src/praisonai/praisonai/auto.py

```
@@ -234,28 +235,34 @@ def _check_openai_available() -> bool:
 
 
 def _get_openai_client(api_key: str = None, base_url: str = None):
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)